.travis.yml

```diff
@@ -19,10 +19,14 @@ matrix:
     - php: 5.5
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
-      env: SYMFONY_VERSION=2.7.*
-    - php: 5.6
-      env: SYMFONY_VERSION=2.8.*
+      env: DEPENDENCIES="symfony/lts:^2"
+    - php: 7.1
+      env: DEPENDENCIES="symfony/lts:^3"
+    - php: 7.1
+      env: STABILITY="dev"
   allow_failures:
+    - php: 7.1
```

Comment:
Remove `php: 7.1` from the conditions matching the allowed failure (matching any PHP version as long as it has `env: STABILITY="dev"`). This way, we have a single place to update when changing the PHP version for the dev job